server/src/test/java/org/apache/calcite/avatica/RemoteDriverMockTest.java

```diff
@@ -127,6 +127,21 @@ private Connection getMockConnection() {
     connection.close();
   }
 
+  @Test public void testTablesAdditionalColumns() throws Exception {
```

Comment:
I'm not convinced of the usefulness of this test. Which part of the production code is actually being exercised? Kinda looks like we're just stubbing a mock, then testing that we stubbed it.

Reply:
It mocks getting the response, but it actually tests the client code in RemoteMeta that handles the response and parses the signature into the ResultSet.